app/src/main/java/org/keepgoeat/presentation/setting/GoalSettingActivity.kt

```diff
@@ -80,6 +82,12 @@ class GoalSettingActivity :
             showKeyboard(it, false)
             viewModel.uploadGoal()
         }
+        binding.etGoal.setOnFocusChangeListener { _, hasFocus ->
+            if (!hasFocus && viewModel.isValidTitle.value)
+                binding.tvGoalTitleLengthGuide.visibility = View.GONE
+            else
+                binding.tvGoalTitleLengthGuide.visibility = View.VISIBLE
+        }
```

Comment:
```suggestion
binding.etGoal.setOnFocusChangeListener { _, hasFocus ->
            if (hasFocus && !viewModel.isValidTitle.value)
                binding.tvGoalTitleLengthGuide.visibility = View.VISIBLE
            else
                binding.tvGoalTitleLengthGuide.visibility = View.INVISIBLE
        }
```

이렇게 수정해보시고 xml에서 tvGoalTitleLengthGuide visibility 속성을 제거해보세용~